src/pynxtools_raman_multiformat/config_file.json

```diff
@@ -0,0 +1,63 @@
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/detector_number": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/gain_setting": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/type": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/power": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/power/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/FABRICATION[device_information]/vendor": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/FABRICATION[device_information]/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/type": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/FABRICATION[device_information]/vendor": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/FABRICATION[device_information]/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/magnification": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/numerical_aperture": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/device_information/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/grating/period": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/grating/period/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/wavelength": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/wavelength/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/attached_to": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/measurement": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/value": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/value/@units": "@eln",
+  "/ENTRY[entry]/SAMPLE[sample]/name": "@eln",
+  "/ENTRY[entry]/SAMPLE[sample]/atom_types": "@eln",
+  "/ENTRY[entry]/USER[user]/address": "@eln",
+  "/ENTRY[entry]/USER[user]/affiliation": "@eln",
+  "/ENTRY[entry]/USER[user]/email": "@eln",
+  "/ENTRY[entry]/USER[user]/name": "@eln",
+  "/ENTRY[entry]/experiment_description": "@eln",
+  "/ENTRY[entry]/experiment_type": "@eln",
+  "/ENTRY[entry]/raman_experiment_type": "@eln",
+  "/ENTRY[entry]/title": "@eln",
+  "/ENTRY[entry]/start_time": "@eln",
+  "/ENTRY[entry]/definition": "@eln",
+  "/ENTRY[entry]/@default": "data",
+  "/ENTRY[entry]/DATA[data]/@signal": "y_values",
+  "/ENTRY[entry]/DATA[data]/y_values": "@data:y_values",
+  "/ENTRY[entry]/DATA[data]/x_values": "@data:x_values",
+  "/ENTRY[entry]/DATA[data]/y_values/@unit": "@attrs:/ENTRY[entry]/data/unit_y",
+  "/ENTRY[entry]/DATA[data]/x_values/@unit": "@attrs:/ENTRY[entry]/data/unit_x",
+  "/ENTRY[entry]/DATA[data]/y_values/@long_name": "@attrs:/ENTRY[entry]/data/longname_y",

```

Comment:
Are you sure you have the `data/longname_y` key in the txt file?

Reply:
I get this @attrs value from the eln_data.yaml file in my case.

Reply:
Ah yea, I think I can figure this out now faster and now what you man and how to handle it. Ill give it a try.